mysql-test/include/restart_mysqld_with_invalid_option.inc

```diff
@@ -0,0 +1,35 @@
+
```

Comment:
Based on the last @georgelorchpercona comment now I recall that the "canonical" way to test the server failure to start is
--source include/shutdown_mysqld.inc
--error 1
--exec $MYSQLD options, including a dedicated error log
grep, remove the dedicated error log (can also be done after start_mysqld.inc)
--source include/start_mysqld.inc

For example, https://github.com/mysql/mysql-server/blob/5.6/mysql-test/suite/innodb/t/innodb-log-file-size-1.test#L209